upgrade/1.4.2/README.md

```diff
@@ -174,6 +174,42 @@ This step will install the `smart-mon` rpm on storage nodes, and reconfigure the
    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
    ```
 
+## Configure NCN nodes without restart
+
+This step will create an imperative CFS session that can be used to configure booted NCN nodes with updated sysctl values.
+
+1. (`ncn-m001#`) Create a new CFS Configuration Entry for the release
+
+```bash
+COMMIT=`kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' 2>/dev/null | yq r -j - 2>/dev/null | jq --arg version "$CSM_RELEASE_VERSION" '. [$version].configuration.commit' | tr -d '"'`
+echo '{
+  "layers": [
+    {
+      "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+      "commit": "COMMIT",
+      "name": "ncn_sysctl",
+      "playbook": "ncn_sysctl.yml"
+    }
+  ]
+}' | sed -e "s/COMMIT/$COMMIT/g" > /tmp/ncn_sysctl.yml.json
+cray cfs configurations update ncn_sysctl --file /tmp/ncn_sysctl.yml.json
+# Cleanup temporary file
+rm /tmp/ncn_sysctl.yml.json
+```   
+
+2. (`ncn-m001#`) Launch CFS to configure NCN sysctl values
```

Comment:
The spacing is wrong, and the list is actually broken however it is hard to tell since each item is enumerated. Each code block needs to be indented, as you can see in the screenshot that none of the code blocks show as a nested item to the list item.
![image](https://github.com/Cray-HPE/docs-csm/assets/7772179/f6b2ca4b-f9e0-4a0f-9076-9e5493e9bad6)

Reply:
Thanks; resolving now.